DESCRIPTION

```diff
@@ -11,15 +11,21 @@ Authors@R: c(person("Jon","Clayden",role=c("cre","aut"),email="code@clayden.org"
              person("Jean-loup","Gailly",role="cph"),
              person("Mark","Adler",role="cph"))
 Imports: Rcpp (>= 0.11.0)
-Suggests: testthat (>= 0.11.0), covr, reportr, shades
-Enhances: oro.nifti, tractor.base
+Suggests: 
+    testthat (>= 0.11.0),
+    covr,
+    reportr,
+    shades,
+    oro.nifti,
+    tractor.base
 LinkingTo: Rcpp
 Description: Provides very fast read and write access to images stored in the
     NIfTI-1 and ANALYZE-7.5 formats, with seamless synchronisation between
     compiled C and interpreted R code. Also provides a simple image viewer, and
     a C/C++ API that can be used by other packages. Not to be confused with
     'RNiftyReg', which performs image registration.
 License: GPL-2
+SystemRequirements: C++11
```

Comment:
This is overkill, as noted in issue #16. I just had a couple of equals signs missing... 😅